apps/emqx_resource/src/emqx_resource.erl

```diff
@@ -45,12 +45,16 @@
 %% todo: replicate operations
 -export([ create/3 %% store the config and start the instance
         , create/4
+        , create/5
         , create_local/3
         , create_local/4
+        , create_local_with_group/4
```

Comment:
We'd better change the old API `create_local/3,4` to add another parameter for group-name